test/guard_SUITE.lfe

```diff
@@ -972,11 +972,11 @@
                                       (=:= (element 2 t) 'integers))
                              (+ (element 3 t) (element 4 t))
                              'true 'error)))
-     (line (test-pat 65 (case ()
-                          (() (andalso (=:= (element 1 t) 'type)
-                                       (=:= (tuple_size t) 4)
-                                       (=:= (element 2 t) 'integers))
-                           (+ (element 3 t) (element 4 t))))))
+     (line (test-pat true (case ()
+                            (() (andalso (=:= (element 1 t) 'type)
+                                         (=:= (tuple_size t) 4)
+                                         (=:= (element 2 t) 'integers)))
+                            ('true (+ (element 3 t) (element 4 t))))))
```

Comment:
I think this line can be removed. 
@rvirding, is this the correct test in this case?